.pipelines/templates/package-helm-chart.yaml

```diff
@@ -0,0 +1,6 @@
+steps: 
+- bash: | 
+    helm dependency update osm-arc
+    helm package --version $(image.tag) --destination $(image.dir) $(repo.chart.path) --debug
```

Comment:
I have created https://github.com/Azure/osm-azure/issues/27 to use git hash for `image.tag`.